lib/signing/types.ts

```diff
@@ -100,7 +101,7 @@ export interface PresentationSignCallBackParams {
   /**
    * The selected credentials to include in the eventual VP as determined by PEX and/or user
    */
-  selectedCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | string)[];
+  selectedCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[];
```

Comment:
Could also be replaced by OriginalVerifiableCredential i think

Reply:
changed